build/docker/build-tools/build-tools.dockerfile

```diff
@@ -2,10 +2,10 @@
 FROM ubuntu:22.04
 ARG ARCH=amd64
 RUN apt-get update && apt-get install -y wget \
-    vim git make gcc upx-ucl 
+    vim git make gcc upx-ucl
 
 RUN if [ "${ARCH}" = "amd64" ]; then apt-get install -y gcc-aarch64-linux-gnu libc6-dev-arm64-cross gcc-arm-linux-gnueabihf ; fi
- 
+
```

Comment:
This dockerfile is auto modified? If merged, it will push the build-tools image again.

Reply:
> This dockerfile is auto modified? If merged, it will push the build-tools image again.


IDE automatically removed some unnecessary whitespace.  I have manually reverted the changes in this pr.